.jenkins/build_docker_images.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Set variable to allow parallel building in the Docker image creation
+export DOCKER_BUILDKIT=1
```

Comment:
This env variable allows you to use buildkit while still retaining the full output from the build (by default docker buildkit either hides it or results in a lot of spammed, repeated progress lines, both don't work well for build systems).

```suggestion
export DOCKER_BUILDKIT=1
export BUILDKIT_PROGRESS=plain
```